src/org/labkey/test/tests/DomainDesignerTest.java

```diff
@@ -674,20 +680,26 @@ public void setPhiLevel() throws Exception
         DomainDesignerPage domainDesignerPage = DomainDesignerPage.beginAt(this, getProjectName(), "exp.materials", sampleSet);
         DomainFormPanel domainFormPanel = domainDesignerPage.fieldProperties(sampleSet);
         DomainFieldRow notPhi = domainFormPanel.addField("notPHI");
-        notPhi.setPHILevel("Not PHI");
+        notPhi.setPHILevel(PropertiesEditor.PhiSelectType.NotPHI);
         DomainFieldRow limitedPHI = domainFormPanel.addField("limitedPHI");
-        limitedPHI.setPHILevel("Limited PHI");
+        limitedPHI.setPHILevel(PropertiesEditor.PhiSelectType.Limited);
         DomainFieldRow fullPHI = domainFormPanel.addField("fullPHI");
-        fullPHI.setPHILevel("Full PHI");
+        fullPHI.setPHILevel(PropertiesEditor.PhiSelectType.PHI);
         DomainFieldRow restrictedPHI = domainFormPanel.addField("restrictedPHI");
-        restrictedPHI.setPHILevel("Restricted PHI");
+        restrictedPHI.setPHILevel(PropertiesEditor.PhiSelectType.Restricted);
 
         domainDesignerPage.clickSaveAndFinish();
         dgen.insertRows(createDefaultConnection(true), dgen.getRows());
         clickAndWait(Locator.linkWithText(sampleSet));
 
         DataRegionTable sampleSetTable = DataRegionTable.findDataRegionWithinWebpart(this,"Sample Set Contents");
         DomainResponse domainResponse = dgen.getDomain(createDefaultConnection(true));
+        Map<String, Object> testColumnProperties = getPropertyPerColumn(domainResponse.getColumns(), "PHI",
+                "notPHI", "limitedPHI", "fullPHI", "restrictedPHI");
+        assertEquals("NotPHI", testColumnProperties.get("notPHI"));
+        assertEquals("Limited", testColumnProperties.get("limitedPHI"));
+        assertEquals("PHI", testColumnProperties.get("fullPHI"));
+        assertEquals("Restricted", testColumnProperties.get("restrictedPHI"));
 
         Map<String, Object> notPHiField = domainResponse.getColumns().stream().filter(a-> a.get("name").equals("notPHI")).findFirst().orElse(null);
         assertEquals("NotPHI", notPHiField.get("PHI"));
```

Comment:
Remove old, duplicate asserts. They have been superseded by the asserts above.

Reply:
Why is that imperative?
I left those in because they get a complete map of the field- which will be a lot better for later debugging than just the derivative/reductive map we created in getPropertyPerColumn.
GPPC does nicely add better logging for the case where the column itself is missing, but it doesn't get us all the state we might want later

Reply:
If one of these checks would fail, the previous, identical assertions would have already cought them. The opposite goes for if they would pass. If you want to dig through the response in a debugger, you already have the `domainResponse`stashed.